src/main/scala/no/ndla/learningpathapi/controller/LearningpathController.scala

```diff
@@ -359,16 +367,19 @@ trait LearningpathController {
 
     post("/", operation(addNewLearningpath)) {
       val newLearningPath = extract[NewLearningPath](request.body)
-      val createdLearningPath: Option[LearningPath] = optLong("copy-from") match {
-        case Some(id) => updateService.newFromExisting(id, newLearningPath, usernameFromHeader)
-        case None => Some(updateService.addLearningPath(newLearningPath, usernameFromHeader))
-      }
+      val learningPath = updateService.addLearningPath(newLearningPath, usernameFromHeader)
+      logger.info(s"CREATED LearningPath with ID =  ${learningPath.id}")
+      halt(status = 201, headers = Map("Location" -> learningPath.metaUrl), body = learningPath)
+    }
 
-      createdLearningPath match {
-        case None => halt(status = 404, body = Error(Error.NOT_FOUND, s"Learningpath with id ${params("copy-from")} not found"))
+    post("/copy/:path_id", operation(copyLearningpath)) {
```

Comment:
Hmm...er enig i at det er fornuftig å ta ibruk egne objekter for input for både ny og kopi, da det ikke er samme valideringsregler. Liker ikke helt at det er et eget endepunkt for `/copy`. Semantikken får meg til å tro at man her legger til en ny læringssti i samlingen som heter `copy`. Hva med å ha dem i samme endepunkt, men å kjøre `extract` til to forskjellige case-classes avhengig om det er en kopi eller ikke?